### PR TITLE
[FIX] website: restore the redirect after lang install

### PR DESCRIPTION
**Pull request by rdeodoo**
Since [1] the redirection supposed to happen after a lang install from the
frontend was broken.
Indeed, the 'url_return' was moved and used before it is actually set.

[1]: 745ef9d

Cheers to @JKE-be which did the same fix in another branch in parallel.

### CODE REVIEW

File: addons/website/tests/test_ui.py [hunk 1]
```diff
@@ -101,9 +101,12 @@ def test_html_editor_scss(self):
 @odoo.tests.tagged('-at_install', 'post_install')
 class TestUiTranslate(odoo.tests.HttpCase):
     def test_admin_tour_rte_translator(self):
-        fr_BE = self.env.ref('base.lang_fr_BE')
-        fr_BE.active = True
-        self.env.ref('website.default_website').language_ids |= fr_BE
+        self.env['res.lang'].create({
+            'name': 'Parseltongue',
+            'code': 'pa_PA',
```
Comment (mart-e) [inline addons/website/tests/test_ui.py:106]: Parseltongue from Paraguay? Maybe 
```suggestion
            'code': 'pa_GB',
```

**Review by mart-e — APPROVED** [review]

File: addons/website/tests/test_ui.py (hunk 1 above)
Comment (rdeodoo) [reply addons/website/tests/test_ui.py]: Makes more sense, even if I am not too sure where this language really comes from :grimacing: 
Changed :+1:

**Comment by rdeodoo** [comment]
@robodoo r+

**Comment by rdeodoo** [comment]
@robodoo rebase-ff

**Pull request closed without merging**